Tomes/Tome.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using Newtonsoft.Json.Linq;
+
+public class Tome {
+	public delegate void OnChanged(JToken oldValue);
+	public delegate void OnDestroy();
+	public delegate void OnAdd(JToken key);
+	public delegate void OnDel(JToken key);
+
+	//
+	public static JToken Conjure(JToken data, JToken root = null) {
+		switch (data.Type) {
+		case JTokenType.Array:
+			return new TomeArray((JArray)data, root);
+		case JTokenType.Object:
+			return new TomeObject((JObject)data, root);
+		default:
+			return new TomeValue((JValue)data, root);
+		}
+	}
+
+	//
+	public static JToken PathValue(JToken data, JArray paths) {
+		JToken value = data;
```

Comment:
Do you need this or could you just use `data` directly? (if so, feel free of course to rename `data` to `value`).